public/example.js

```diff
@@ -0,0 +1 @@
+console.log('test of next.js script tag.');
```

Comment:
Please remove

Reply:
But the point of this PR is to add example code to `markdoc-starter`.